phpBB/adm/style/acp_logs.html

```diff
@@ -77,14 +77,6 @@
 	</fieldset>
 <!-- ENDIF -->
 
-<!-- IF .log and S_CLEARLOGS -->
```

Comment:
no, you can simply add the html with firebug?
If you want to prevent deleting them, you have to delete the php code that processes the form